rm_fsm:
  robot_type: "hero"
  control_param:
    accel_x: 10.0
    accel_y: 10.0
    accel_angular: 10.0
    brake_multiple: 2
    expect_shoot_hz: 5
    safe_shoot_hz: 2
    safe_shoot_speed: 10
    gimbal_error_limit: 0.1745
    rc_param:
      coefficient_x: 3.5
      coefficient_y: 3.5
      coefficient_angular: 6.0
      coefficient_yaw: 12.56
      coefficient_pitch: 12.56
    pc_param:
      coefficient_x: 3.5
      coefficient_y: 3.5
      coefficient_angular: 1.0
      coefficient_yaw: 12.56
      coefficient_pitch: 12.56
      normal_critical_speed: 1
      burst_critical_speed: 2
      normal_angular: 1
      burst_angular: 2
  power_limit:
    danger_surplus: 20.0
    roll_back_buffer: 55.0
    coeff: 0.3
    multiple: 15.0